scoreboard/models.py

```diff
@@ -122,15 +148,20 @@ def from_xlog(self, source, xlog_dict):
 
         # add conducts (does it make sense to do this for non-ascended games?)
```

Comment:
It makes sense to do this for non-ascended games because we may at some point want to poach the "games with all [12 vanilla] conducts broken" metric from Junethack.